auto-configurations/models/spring-ai-autoconfigure-model-vertex-ai/src/main/java/org/springframework/ai/model/vertexai/autoconfigure/gemini/VertexAiGeminiChatProperties.java

```diff
@@ -35,7 +36,10 @@ public class VertexAiGeminiChatProperties {
 
 	/**
 	 * Vertex AI Gemini API generative options.
+	 * 
+	 * @author Hyunsang Han
```

Comment:
```suggestion
```

Reply:
Please move the @author tag to the class level.

```java
  * @author Christian Tzolov
  * @author Hyunsang Han
  * @since 0.8.0
  */
@ConfigurationProperties(VertexAiGeminiChatProperties.CONFIG_PREFIX)
public class VertexAiGeminiChatProperties {
```

Reply:
Oh, I missed it. Thanks!